### ISSUE TITLE
Portar Módulo 03

### ISSUE THREAD

**Issue by flaviovdf** [deep-ufmg/praticas-versao-pytorch#3]
- [ ] Criar testes de unidade, não tem testes
- [x] Código em Pytorch
- [ ] Uma tarefa de implementar uma convolução do zero
- [ ] Guiar o aluno
  - [ ] Baselines (random, majority, talvez um scikit-learn simples)
  - [ ] Explicar melhor as bases. Balanceamento de classes, número de features, se precisa normalizar ou não
